base/steer/FairTSBufferFunctional.h

```diff
@@ -32,9 +32,13 @@ class TTree;
  * The method TimeOut is used to break the processing if for example always the same data is requested.
  */
 
-class BinaryFunctor : public std::binary_function<FairTimeStamp*, double, bool>
+class BinaryFunctor
 {
   public:
+    using first_argument_type = FairTimeStamp*;
+    using second_argument_type = double;
+    using result_type = bool;
+
```

Comment:
I wonder, whether those are actually used? Can we mark them deprecated?

Reply:
It looks like they are not used, but before deprecating this on such a fine-grained level, I would rather just deprecate/remove the whole `BinaryFunctor` type and replace it with `std::function<bool(FairTimeStamp*,double)>`.

Reply:
Okay. Let's postpone this for another time.